vnext/Microsoft.ReactNative/Utils/LocalBundleReader.cpp

```diff
@@ -15,6 +15,44 @@
 
 namespace Microsoft::ReactNative {
 
+std::string GetBundleFromEmbeddedResource(winrt::hstring str) {
+  winrt::Windows::Foundation::Uri uri(str);
+  auto moduleName = uri.Host();
+  auto path = uri.Path();
+  // skip past the leading / slash
+  auto resourceName = path.c_str() + 1;
+  std::string script;
+
+  if (auto hmodule = GetModuleHandle(moduleName != L"" ? moduleName.c_str() : nullptr)) {
```

Comment:
>if [](http://example.com/codeflow?start=2&length=2)

nit: It would be nice to avoid the deep nesting of `if` statements. Can it be flatten out by checking each condition and then escaping earlier if it fails?